content/format-reduction/index.md

```diff
@@ -0,0 +1,56 @@
+---
+rfc:
+start_date: 2018-07-26
+pr:
+status: draft
+---
+
+# Format reduction
+
+## Summary
+
+This RFC proposes to reduce the amount of representations in Registers.
+
+
+## Motivation
+
+The Registers data model and its REST API is still in motion. The amount of
+formats suggested in the specification and implemented in the reference
+implementation increase unnecessarily the complexity of evolving them. For
+example, we want to evolve who the schema fits into the data model and how it
+is surfaced to the user.
+
+
+## Explanation
+
+The current suggestion in the spec is to implement JSON, YAML, CSV, TSV, HTML
+and Turtle. Although none of them are required, the reference implementation
+has implemented all of them.
+
+There are a few issues to notice:
+
+The Turtle implementation is not quite
+right and the specification is not mature enough to make different
+implementors implement the same thing by following the specification alone.
+
+The YAML representation offers no value on top of the JSON one.
+
+The CSV and TSV are virtually the same.
+
+The HTML representtion has been proven confusing for users in the reference
+implementation and has diverged from being the exact equivalent to the other
+representations so, failing its purpose to be another representation for these
+resources.
+
+The current usage of Registers we can determine that users as per 2018 favour
+JSON (89.9%) and a small fraction benefit from a tabular form (CSV 3.5%, TSV
+1.9%). Turtle (1.6%) offers a distinct set of features but as mentioned above
+is not mature enough and it is misleading users.
+
+In light of that, the suggestion is to remove all formats but JSON and CSV.
+And bring back RDF when the specification has a mature definition of how to
```

Comment:
This sentence doesn't flow naturally to me - can you drop the "and" and change it to "We should bring back RDF when..."